src/main/resources/data/ancientology/recipes/copper_nugget_from_ingot.json

```diff
@@ -0,0 +1,19 @@
+{
+	"type": "minecraft:crafting_shaped",
```

Comment:
This could just be a shapeless recipe, and I agree with Matty on this with the formatting being weird, just use data gen and avoid duplicate recipes